blender/LilySurfaceScraper/Scrapers/PolyHavenTextureScraper.py

```diff
@@ -87,21 +104,24 @@ def fetchVariantList(self, url):
                 for fmt, map_data in formats.items():
                     variant_data[(res, fmt)][map_type] = map_data['url']
 
-        variant_data = [ (*k, v) for k, v in variant_data.items() ]
-        variant_data.sort(key=lambda x: (x[1].zfill(3), x[0]))
-        variants = [ f"{res} ({fmt})" for res, fmt, _ in variant_data ]
+        variant_data = [(*k, v) for k, v in variant_data.items()]
+        variant_data.sort(key=lambda x: self.sortTextWithNumbers(f"{x[0]} ({x[1]})"))
```

Comment:
I don't think this is equivalent

Reply:
my version handles numbers in text better, so it sorts like this
![image](https://user-images.githubusercontent.com/16857861/126909849-4b264400-fbde-4fcc-ae3a-09e54f36c1b4.png)
instead of like this
![image](https://user-images.githubusercontent.com/16857861/126909856-c2a217e9-97e3-4729-b988-50705743cc82.png)

Reply:
Mine was sorting the resolutions correctly as well thanks to `.zfill(3)` and I prefer to have the variants sorted by extension first, only then by resolution. It makes more sense imho because for a given project, one will usually stick to the same file format for all objects, but adapt the resolution to the size of the object (to avoid wasting details).

Reply:
I can change it to sort extensions first then resolutions.
the problem with sorting after a `.zfill(3)` is that it's still sorting a string, so it cant handle numbers bigger than 10 correctly (since for example `6` is smaller than `k` so it puts it first)

Reply:
is this better?
![image](https://user-images.githubusercontent.com/16857861/126912393-548ef8fa-2767-4f61-9860-e816fa85a69d.png)